### PR TITLE
Register link master

### PR DESCRIPTION
**Pull request by davidcarroll**
This is the work Karen and I have been working on for the FPU registrations. Please review and publish to production when you get a chance. And let Karen know when it is done so she can publish the blog post.

### CODE REVIEW

File: CmsData/Email/EmailReplacements.cs [hunk 1]
```diff
@@ -324,6 +327,10 @@ private string DoReplaceCode(string code, Person p, PayInfo pi = null, EmailQueu
                     if (code.StartsWith("{orgmember:", StringComparison.OrdinalIgnoreCase))
                         return OrgMember(code, emailqueueto);
 
+                    if(code.StartsWith("{orgbarcode:"))
+                        return string.Format("<img src='{0}' />",
```
Comment (drmohundro) [inline CmsData/Email/EmailReplacements.cs]: Did you want new code to take advantage of C#6 string interpolation? Or hold off on it for this commit?

File: CmsWeb/Content/touchpoint/lib/froala-editor/js/plugins/special_links.js [hunk 2]
```diff
@@ -97,6 +97,7 @@
             '<option value="sendlink2">Send Link 2</option>' +
             '<option value="supportlink">Support Link</option>' +
             '<option value="votelink">Vote Link</option>' +
+            '<option value="masterlink">Master Link (NOT SUPPORTED YET)</option>' +
```
Comment (drmohundro) [inline CmsWeb/Content/touchpoint/lib/froala-editor/js/plugins/special_links.js]: Is this still *NOT SUPPORTED YET*?

File: CmsData/Email/EmailReplacements.cs (hunk 1 above)
Comment (davidcarroll) [inline CmsData/Email/EmailReplacements.cs]: We can leave this for now and do it all in a big refactoring project. I don't mind using it for new code from now on.

File: CmsWeb/Content/touchpoint/lib/froala-editor/js/plugins/special_links.js (hunk 2 above)
Comment (davidcarroll) [inline CmsWeb/Content/touchpoint/lib/froala-editor/js/plugins/special_links.js]: Yes, until Karen documents it.

**Comment by drmohundro** [comment]
I added a note about if we're using C# string interpolation now or not as I saw a few places that didn't have that.

Also, quick question - I noticed some of the razor code was using 2 space indentation (versus 4 space indentation). Did we change that or should it be 4 spaces?

**Comment by davidcarroll** [comment]
I don't recall changing anything, but we should make it consistent as a matter of policy.

**Comment by drmohundro** [comment]
Okay, we can check on that later.

**Pull request merged**